networkx/algorithms/approximation/steinertree.py

```diff
@@ -0,0 +1,75 @@
+from itertools import combinations, chain
+
+from networkx.utils import pairwise, not_implemented_for
+import networkx as nx
+
+__all__ = ['metric_closure', 'steiner_tree']
+
+
+@not_implemented_for('directed')
+def metric_closure(G, weight='weight'):
+    """  Return the metric closure of a graph.
+
+    The metric closure of a graph G is the complete graph in which each edge is
```

Comment:
When referring to a mathematical variable like *G* in this case, please use asterisks around the variable now so that the variable is rendered in italics, like `*G*`, rendered as *G*. Apply this advice throughout this file!